src/mlpack/tests/recurrent_network_test.cpp

```diff
@@ -15,6 +15,10 @@
 #include <mlpack/methods/ann/layer/layer.hpp>
 #include <mlpack/methods/ann/rnn.hpp>
 #include <mlpack/core/data/binarize.hpp>
+#include <mlpack/core/optimizers/rmsprop/rmsprop.hpp>
+#include <cmath>
```

Comment:
This should already be included by `prereqs.hpp` so I think there is no need to include it here.

Reply:
RMSProp is not included somehow, I was getting a compile error. Therefore I had to add it.
Removed `cmath`